.gitignore

```diff
@@ -6,3 +6,5 @@ vendor/
 .DS_Store
 go.sum
 localenvoyconfig.yaml
+securelocalenvoyconfig.yaml
```

Comment:
This is ok for this PR, but in the future i'd prefer to not have a process that poops random files into the root of the repository. Please take this as a todo to try to eliminate these in the future.